ui/src/app/views/quick-tour-modal/component.ts

```diff
@@ -79,9 +79,9 @@ export class QuickTourModalComponent implements OnInit {
       'A Workspace is your place to store and analyze data for a specific project. ' +
       'You can share this Workspace with other users, allowing them to view or edit ' +
       'your work. The dataset referenced by a workspace is in ' +
-      '<a href="https://www.ohdsi.org/data-standardization/the-common-data-model/" ' +
-      'target="_blank">OMOP common data model</a> format. ' +
-      '<a href="https://www.ohdsi.org/past-events/2017-tutorials-omop-common-data' +
+      '<a class="link" href="https://www.ohdsi.org/data-standardization/the-common-data-model/" ' +
+      'target="_blank" data-foo="bar">OMOP common data model</a> format. ' +
```

Comment:
`data-foo="bar"` ?

Reply:
oops that was from our debugging!  good catch